scenarios2/10_Lilith.cfg

```diff
@@ -1058,44 +1062,46 @@
 
 #endif
 #ifdef HARD
-        {GENERIC_UNIT 5 "Elvish Archer" 20 17}
-        {GENERIC_UNIT 5 "Elvish Archer" 21 17}
-        {GENERIC_UNIT 5 "Elvish Archer" 22 17}
-        {GENERIC_UNIT 5 "Elvish Archer" 23 17}
-        {GENERIC_UNIT 5 "Elvish Archer" 24 17}
-        {GENERIC_UNIT 5 "Elvish Shaman" 21 18}
-        {GENERIC_UNIT 5 "Elvish Shaman" 22 18}
-        {GENERIC_UNIT 5 "Elvish Shaman" 23 18}
-        {GENERIC_UNIT 5 "Elvish Fighter" 21 19}
-        {GENERIC_UNIT 5 "Elvish Fighter" 22 19}
-        {GENERIC_UNIT 5 "Elvish Fighter" 23 19}
-        {GENERIC_UNIT 5 "Elvish Archer" 21 20}
-        {GENERIC_UNIT 5 "Elvish Archer" 22 20}
-        {GENERIC_UNIT 5 "Elvish Archer" 23 20}
-        {GENERIC_UNIT 5 "Elvish Archer" 20 19}
-        {GENERIC_UNIT 5 "Elvish Archer" 24 19}
-        {GENERIC_UNIT 5 "Elvish Archer" 19 13}
-        {GENERIC_UNIT 5 "Elvish Archer" 19 14}
-        {GENERIC_UNIT 5 "Elvish Archer" 19 15}
-        {GENERIC_UNIT 5 "Elvish Archer" 19 16}
-        {GENERIC_UNIT 5 "Elvish Archer" 19 17}
-        {GENERIC_UNIT 5 "Elvish Archer" 19 18}
-        {GENERIC_UNIT 5 "Elvish Archer" 19 19}
-        {GENERIC_UNIT 5 "Elvish Archer" 25 13}
-        {GENERIC_UNIT 5 "Elvish Archer" 25 14}
-        {GENERIC_UNIT 5 "Elvish Archer" 25 15}
-        {GENERIC_UNIT 5 "Elvish Archer" 25 16}
-        {GENERIC_UNIT 5 "Elvish Archer" 25 17}
-        {GENERIC_UNIT 5 "Elvish Archer" 25 18}
-        {GENERIC_UNIT 5 "Elvish Archer" 25 19}
-        {GENERIC_UNIT 5 "Elvish Fighter" 18 14}
-        {GENERIC_UNIT 5 "Elvish Fighter" 18 15}
-        {GENERIC_UNIT 5 "Elvish Fighter" 18 17}
-        {GENERIC_UNIT 5 "Elvish Fighter" 18 18}
-        {GENERIC_UNIT 5 "Elvish Fighter" 26 14}
-        {GENERIC_UNIT 5 "Elvish Fighter" 26 15}
-        {GENERIC_UNIT 5 "Elvish Fighter" 26 17}
-        {GENERIC_UNIT 5 "Elvish Fighter" 26 18}
+        # Need to do something with these.  Is even L3 enough (so many die when Argan arrives)?  
+        # Leave them as L1, but mix in a bunch of really awesome leadership to make enemies harder but still only 8XP?
```

Comment:
Yeah, sounds like a bunch of Elvish Overlords, maybe with some leadership-like bonuses would help.

Reply:
Done.  Also changed normal, but didn't give them leadership6 there.